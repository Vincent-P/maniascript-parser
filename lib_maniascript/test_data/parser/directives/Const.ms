#Const C_HudModulePath "Obstacle/UI.Module.Gbx"
