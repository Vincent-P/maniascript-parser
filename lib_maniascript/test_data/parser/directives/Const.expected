NODE_ROOT 0..47 {
  NODE_CONST 0..47 {
    TOKEN_CONST("#Const") 0..6
    TOKEN_WHITESPACE(" ") 6..7
    NODE_IDENTIFIER 7..22 {
      TOKEN_IDENT("C_HudModulePath") 7..22
    }
    TOKEN_WHITESPACE(" ") 22..23
    NODE_STRING 23..47 {
      TOKEN_STRING_START("\"") 23..24
      TOKEN_STRING_CONTENT("Obstacle/UI.Module.Gbx") 24..46
      TOKEN_STRING_END("\"") 46..47
    }
  }
}